lib/launcher.js

```diff
@@ -430,7 +438,7 @@ class Launcher {
                     opts.throwHttpErrors = false
                 }
                 // Use a HEAD request to minimise data transfer
-                const res = await got.head(pollUrl, opts)
+                const res = await got.default.get.head(pollUrl, opts)
```

Comment:
```
Failed to poll NR on  http://127.0.0.1:12081/ TypeError: got.default.get.head is not a function
    at statusPoll (/opt/share/projects/flowforge/dev-env/packages/nr-launcher/lib/launcher.js:441:51)
    at Timeout._onTimeout (/opt/share/projects/flowforge/dev-env/packages/nr-launcher/lib/launcher.js:512:17)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
```

Reply:
`yeah, seems I had multiple cursors active when I modified the code. I corrected the `got.default.get` (and missed the fact I'd messed up the others.

Pushed an update to revert the 2 places that should not have been modified